core/templates/pages/exploration-editor-page/services/exploration-rights-backend-api.service.ts

```diff
@@ -42,7 +42,9 @@ export class ExplorationRightsBackendApiService {
   ) {}
 
   async makeCommunityOwnedPutData(
-      explorationId: string, version: number, makeCommunityOwned: boolean):
+      explorationId: string,
+      version: number,
+      makeCommunityOwned: boolean):
```

Comment:
If all of the arguments can fit on a single line, let's keep them on a single line! If the linter says it's too long, then we can put arguments on their own lines like you've done here.

Reply:
when i put them like this  :
`async makeCommunityOwnedPutData(explorationId: string,  version: number, makeCommunityOwned: boolean)` 
linter is gaving error too long. 

when I put them like this :
`async makeCommunityOwnedPutData(        
                                                                                                                           
    explorationId: string, version: number,  makeCommunityOwned: boolean)`
linter is accepting them

`last method is used above`

`which one now should I prefer ?`

Reply:
Let's keep it the way it was before! 
```
async makeCommunityOwnedPutData(
    explorationId: string, version: number,  makeCommunityOwned: boolean)`
```

Reply:
okay

Reply:
Done